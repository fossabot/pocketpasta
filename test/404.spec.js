import { shallowMount } from '@vue/test-utils';
import component from '@/components/error/404.vue';

const factory = () => shallowMount(component, {});

describe('404', () => {
  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
  test('head', () => {
    expect(component.head()).toBeTruthy();
  });
});
