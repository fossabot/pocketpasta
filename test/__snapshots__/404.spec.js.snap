// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 renders properly 1`] = `
"<div id=\\"message\\" class=\\"card bg-light card-body\\"><h2>404</h2> <h1>Page Not Found</h1> <p>
    The specified file was not found on this website. Please check the URL for
    mistakes and try again.
  </p> <nuxt-link to=\\"/\\" class=\\"btn btn-primary\\">Home page</nuxt-link></div>"
`;
